.github/dependabot.yaml

```diff
@@ -1,6 +1,9 @@
 version: 2
 updates:
-  - directory: '/'
+  - commit-message:
+      prefix: chore
+      include: scope
+    directory: '/'
     package-ecosystem: github-actions
```

Comment:
Also `open-pull-requests-limit: 99`?

Reply:
I found this kind of outlandish. If we ever get there, we have a big big problem. But alright, I'll add it!